examples/vl6180x_historytest.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-FileCopyrightText: 2022 Jonas Schatz
+# SPDX-License-Identifier: MIT
+
+# Demo of reading the range from the history buffer of the VL6180x
+# distance sensor
+
+import time
+from typing import List
+
+import board
+import busio
+
+import adafruit_vl6180x
+
+
+# Create I2C bus.
+i2c = busio.I2C(board.SCL, board.SDA)
+
+# Create sensor instance.
+sensor = adafruit_vl6180x.VL6180X(i2c)
+
+# Starting continuous mode
+print("Starting continuous mode")
+sensor.start_range_continuous()
+
+# Main loop prints the ranges every 0.01 seconds for about 5 seconds
+# You should see changes 'ripple through' the history array
+for _ in range(500):
+    # Read the range in millimeters and print it.
+    ranges_mm: List[int] = sensor.ranges_from_history
```

Comment:
See comment in above!

Reply:
If you want to let people know the output is a list, I think the comment above that line is a good candidate :)